remake.yml

```diff
@@ -10,13 +10,35 @@ packages:
   - readr
   - stringr
   - purrr
+  - tools
 
 targets:
   all:
     depends: 3_visualize/out/figure_1.png
 
+  # download all the data
+  1_fetch/out/site_data/nwis_01427207_data.csv:
+    command: download_nwis_site_data(filepath=target_name)
+  1_fetch/out/site_data/nwis_01432160_data.csv:
+    command: download_nwis_site_data(filepath=target_name)
+  1_fetch/out/site_data/nwis_01435000_data.csv:
+    command: download_nwis_site_data(filepath=target_name)
+  1_fetch/out/site_data/nwis_01436690_data.csv:
+    command: download_nwis_site_data(filepath=target_name)
+  1_fetch/out/site_data/nwis_01466500_data.csv:
+    command: download_nwis_site_data(filepath=target_name)
+
+  input_data_table:
+    command: make_table('1_fetch/out/site_data')
+    depends:
+        - "1_fetch/out/site_data/nwis_01427207_data.csv"
+        - "1_fetch/out/site_data/nwis_01432160_data.csv"
+        - "1_fetch/out/site_data/nwis_01435000_data.csv"
+        - "1_fetch/out/site_data/nwis_01436690_data.csv"
+        - "1_fetch/out/site_data/nwis_01466500_data.csv"
+
   site_data:
-    command: download_nwis_data()
+    command: combine_csvs(input_data_table, I("1_fetch/out/site_data"))
```

Comment:
If you do switch to a task table, both `input_data_table` and `site_data` could be the products of *combiners*, which would get specified in `create_task_makefile`.